contracts/test/sdk/IpcContract.t.sol

```diff
@@ -40,13 +40,12 @@ contract RecorderIpcExchange is IpcExchange {
     }
 
     function _handleIpcResult(
-        IpcEnvelope storage original,
+        IpcEnvelope storage,
         IpcEnvelope memory result,
         ResultMsg memory resultMsg
     ) internal override {
         require(!shouldRevert, "revert requested");
         console.log("handling ipc result");
-        require(keccak256(abi.encode(original)) == keccak256(abi.encode(lastEnvelope)));
```

Comment:
`lastEnvelope` is only saved on incoming calls, so this check didn't make sense.